src/components/NotificationContainer.tsx

```diff
@@ -139,19 +149,49 @@ function NotificationsContainer() {
     lastShownNotification.current = latestNotificationItem
   }
 
-  return (
+  const showDialog = React.useCallback(() => {
+    setDialogOpen(true)
+  }, [setDialogOpen])
+
+  const closeDialog = React.useCallback(() => {
+    closeNotification()
+    setDialogOpen(false)
+  }, [setDialogOpen, closeNotification])
+
+  const dialog = (
+    <Dialog fullScreen open={dialogOpen} onClose={closeDialog} TransitionComponent={FullscreenDialogTransition}>
+      <DialogBody
+        top={<MainTitle onBack={closeDialog} title={"Error"} />}
+        actions={
+          <DialogActionsBox>
+            <ActionButton autoFocus icon={<CheckIcon />} onClick={closeDialog} type="primary">
```

Comment:
Let's drop the `<CheckIcon />`, too. I think it's better to not use any icon here, as there is no real action involved.